### DISCUSSION TITLE
Github bridge push results in zero exports

### DISCUSSION THREAD

**Discussion by sunzoje** [git-bug/git-bug#1026] [Q&A] [answered]
I'm on `55e4046b`. I've added gitlab and github bridges. I pull issues from gitlab with success. When pushing to github bridge, I get `exported 0 issues with github bridge`.

Is there way to debug and see why I am not able to export to github?

**Comment by smoyer64** [comment, marked as answer]
You can pull issues from both GitHub and GitLab bridges and identities will be created for all the users that created issues.  For obvious security reasons, you can only push issues through the bridge for your identity (or one of your identities).  Identities now include a field for the GPG key and `git-bug` will eventually also sign commits.

Note that on both GitHub and GitLab, you can't create issues and comments for another user's identity, even if you're an owner/administrator of the target repository.  Even if a Git vendor chose to allow you to impersonate people, you'd never be able to sign commits as them.

**Reply by MichaelMure** [reply]
Also, because pushing issues from one bridge to another would be barely useful (you can't even push a comment on an issue you are not the author because you can't create it to begin with), this is explicitly blocked in git-bug.

**You can only push issues to a bridge that have been pulled from the same bridge, or created in git-bug and not associated with a bridge yet.**
